screens/product/product.css

```diff
@@ -0,0 +1,53 @@
+@import url(/css/global.css);
+
+
+body {
+  display: grid;
+  grid-template-areas:
+    "header  header "
+    "filters products";
+  grid-template-columns: 28.8rem 1fr;
+
+}
+
+
+.filters {
+  grid-area: filters;
+  background-color: var(--auth-background);
+  position: sticky;
+  top: 8.8rem;
+  bottom: 0;
+  height: calc(100vh - 8.8rem);
+  overflow-y: auto;
+}
+
+.filter-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
```

Comment:
you can make  class for it. since, it repeat on your code

Reply:
okay i will refactor.

Reply:
Done